src/graceful/resources/base.py

```diff
@@ -241,20 +241,32 @@ def require_params(self, req):
                 try:
                     if param.many:
                         # params with "many" enabled need special care
-                        params[name] = param.container(
-                            req.get_param_as_list(name, param.value) or
-                            [param.default and param.value(param.default)]
-                        )
+                        values = req.get_param_as_list(
+                            # note: falcon allows to pass value handler using
+                            #       `transform` param so we do not need to
+                            #       iterate through list manually
+                            name, param.validated_value
+                        ) or [
+                            param.default and
+                            param.validated_value(param.default)
```

Comment:
Hmm, do we need validate default param here? I mean, we can do it once during ``__init__``